draft-ietf-mls-protocol.md

```diff
@@ -944,12 +943,12 @@ provide some public information about a user. KeyPackage
 structures provide information about a client that any existing
 member can use to add this client to the group asynchronously.
 
-A KeyPackage object specifies a ciphersuite that the client
-supports, as well as providing a public key that others can use
-for key agreement. The client's identity key can be updated
-throughout the lifetime of the group by sending a new KeyPackage
-with a new identity; the new identity MUST be validated by the
-authentication service.
+A KeyPackage object specifies a ciphersuite that the client supports, as well as
+providing a public key that others can use for key agreement. The client's
+identity key can be updated throughout the lifetime of the group by sending a
+new KeyPackage with a new Credential. However, the identity MUST be the same in
+both Credentials and the new Credential MUST be validated by the authentication
+service.
```

Comment:
This doesn't necessarily make sense because not all credentials have an `identity`